pkg/clusteragent/admission/controllers/webhook/controller_base.go

```diff
@@ -74,14 +108,14 @@ type MutatingWebhook interface {
 // the config one. The reason is that the volume mount for the APM socket added
 // by the config webhook doesn't always work on Fargate (one of the envs where
 // we use an agent sidecar), and the agent sidecar webhook needs to remove it.
-func mutatingWebhooks(wmeta workloadmeta.Component) []MutatingWebhook {
+func mutatingWebhooks(wmeta workloadmeta.Component, rcClient *rcclient.Client, stopCh <-chan struct{}, clusterName string) []MutatingWebhook {
 	webhooks := []MutatingWebhook{
 		config.NewWebhook(wmeta),
 		tagsfromlabels.NewWebhook(wmeta),
 		agentsidecar.NewWebhook(),
 	}
 
-	apm, err := autoinstrumentation.GetWebhook(wmeta)
+	apm, err := autoinstrumentation.GetWebhook(wmeta, rcClient, stopCh, clusterName)
```

Comment:
Same question as below

rcClient, stopCh and clustName are passed down to mutating webhooks, but I don't see they are used anywhere. 

Are you planning to use them in a future PR?